static/admin/config.yml

```diff
@@ -158,8 +158,6 @@ collections:
         name: "image"
         widget: "image"
         hint: "The logo/image for this year's competition."
-        media_folder: ""
```

Comment:
Setting `path` on a collection defaults `media_folder` to an empty string:
https://www.netlifycms.org/docs/beta-features/#folder-collections-media-and-public-folder